### DISCUSSION TITLE
skip command.

### DISCUSSION THREAD

**Discussion by yumir60** [jagrosh/MusicBot#1408] [Questions] [answered]
I have a question. When using the skip command, it asks for votes, we can skip this voting by using the modskip command, but I think only admins can use this command. How do I make normal members skip songs without starting a vote using just the skip command?

**Comment by yumir60** [comment, marked as answer]
completely my stupidity, I saw the setskip command while translating the bot to Turkish.

**Comment by sblmbb** [comment]
Do you put this into the config file? There should be option for skip without voting.. we are small friend only discord server, with people afk in the channels constantly.

**Reply by yumir60** [reply]
just try setskip 0

In this way, no matter who, the music will be skipped after 1 person uses the skip command.
